include/aarch64/gpio.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _GPIO_H_
+#define _GPIO_H_
+
+struct resource_t;
+
+void delay(int64_t count);
```

Comment:
Please don't expose it. You don't need to.

Reply:
Done.